specification/domainservices/resource-manager/Microsoft.AAD/stable/2022-09-01/domainservices.json

```diff
@@ -0,0 +1,1151 @@
+        }
+      }
+    },
+    "DomainSecuritySettings": {
+      "description": "Domain Security Settings",
+      "properties": {
+        "ntlmV1": {
+          "type": "string",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "default": "Enabled",
+          "description": "A flag to determine whether or not NtlmV1 is enabled or disabled.",
+          "x-ms-enum": {
+            "name": "NtlmV1",
+            "modelAsString": true
+          }
+        },
+        "tlsV1": {
+          "type": "string",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "default": "Enabled",
+          "description": "A flag to determine whether or not TlsV1 is enabled or disabled.",
+          "x-ms-enum": {
+            "name": "TlsV1",
+            "modelAsString": true
+          }
+        },
+        "syncNtlmPasswords": {
+          "type": "string",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "default": "Enabled",
+          "description": "A flag to determine whether or not SyncNtlmPasswords is enabled or disabled.",
+          "x-ms-enum": {
+            "name": "SyncNtlmPasswords",
+            "modelAsString": true
+          }
+        },
+        "syncKerberosPasswords": {
+          "type": "string",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "default": "Enabled",
+          "description": "A flag to determine whether or not SyncKerberosPasswords is enabled or disabled.",
+          "x-ms-enum": {
+            "name": "SyncKerberosPasswords",
+            "modelAsString": true
+          }
+        },
+        "syncOnPremPasswords": {
+          "type": "string",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "default": "Enabled",
+          "description": "A flag to determine whether or not SyncOnPremPasswords is enabled or disabled.",
+          "x-ms-enum": {
+            "name": "SyncOnPremPasswords",
+            "modelAsString": true
+          }
+        },
+        "kerberosRc4Encryption": {
+          "type": "string",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "default": "Enabled",
+          "description": "A flag to determine whether or not KerberosRc4Encryption is enabled or disabled.",
+          "x-ms-enum": {
+            "name": "KerberosRc4Encryption",
+            "modelAsString": true
+          }
+        },
+        "kerberosArmoring": {
+          "type": "string",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "default": "Disabled",
+          "description": "A flag to determine whether or not KerberosArmoring is enabled or disabled.",
+          "x-ms-enum": {
+            "name": "KerberosArmoring",
+            "modelAsString": true
+          }
+        },
+        "ldapSigning": {
```

Comment:
>ldapSigning [](http://example.com/codeflow?start=9&length=11)

Consider more direct naming, e.g. maybe something more like signingMethod: { LDAP, AAD }

Reply:
There aren't any different singing options.  LDAP Signing is either on or off.

Reply:
@mentat9 Let me know if this breaking for you.  We discussed the naming of these properties and are comfortable with the naming because of how we plan to use them now and in the future.